src/shilpa/Shilpa_Assignment_17/FrequencyOfNum.java

```diff
@@ -0,0 +1,38 @@
+/*Program 4: From given numbers, return the frequency of target number.
+input : 10,23,23,44,23,10,23,4,23
+target number : 23         
+output : Freq of 23 is 4
+*/
+package shilpa.Shilpa_Assignment_17;
+
+import java.util.Scanner;
+
+public class FrequencyOfNum {
+	int cnt;
+
+	int getFreqOfNumber(int[] numbers, int targetno) {
+		for (int index = 0; index < numbers.length; index++) {
+			if (targetno == numbers[index])
+				cnt++;
```

Comment:
Here,You should return true or false, You are not checking that if cnt >0 then that number is present in array.
and we have have to check that number present in array or not which means you have should return true or false